covalent_ecs_plugin/ecs.py

```diff
@@ -187,10 +183,10 @@ async def submit_task(self, task_metadata: Dict, identity: Dict) -> Any:
 
         # Register the task definition
         self._debug_log("Registering ECS task definition...")
-        self.ecs_task_family_name = f"{dispatch_id}-{node_id}"
+        self._ecs_task_family_name = f"{dispatch_id}-{node_id}"
```

Comment:
Given this is not used anywhere else @FyzHsn let's just make this a local variable
```suggestion
ecs_task_family_name = f"{dispatch_id}-{node_id}"
```

Reply:
Ah I thought about that @AlejandroEsquivel but that value is used later on in `get_status`.